src/components/Education.jsx

```diff
@@ -0,0 +1,54 @@
+const Education = ({ userInfo, changeData }) => {
+  return (
```

Comment:
You can optimize it even further, for that create a text input component in react which take placeholder text, name and value as props. Then create a big JSON object in seperate file like:

```
{
    "education": [
        {"placeholder": "1.Add Institute Name", name: "instituteName1", value: "instituteName1"},
        {"placeholder": "1.Add Course Name", name: "courseName1", value: "courseName1"},
        .
        .
        .
    ]
}

```

And iterate over it to create multiple input field using .map method